### PR TITLE
Deprecation notes and OpenShift version for templates and APB

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: assemblies/assembly_openshift-ansible-playbook/main.adoc [hunk 1]
```diff
@@ -32,6 +33,11 @@ parameters as necessary. A {PRODUCT} Ansible Playbook for the Automation Broker
 `ReadWriteMany` mode is not supported on OpenShift Online and OpenShift Dedicated.
 ====
 
+[NOTE]
+====
+Since {PRODUCT} version 7.5, support for {OPENSHIFT} 3.x, including installation using all templates and using the Automation Broker (Ansible Playbook) is deprecated. New features might not be added, and this functionality will be removed in a future release.
```
Comment (sterobin) [inline assemblies/assembly_openshift-ansible-playbook/main.adoc]: @mramendi Just this one comment. I didn't take the time to look at every rendered output, just a couple.

The current wording is a tad rough to my ear. I suggest the following or similar for all:

> Beginning with {PRODUCT} version 7.5, support for {OPENSHIFT} 3.x is deprecated, including installation using all templates and using the Automation Broker (Ansible Playbook). New features might not be added, and this functionality will be removed in a future release.

File: assemblies/assembly_openshift-freeform/main.adoc [hunk 2]
```diff
@@ -25,6 +26,16 @@ ifdef::PAM[]
 ** {Central} Monitoring requires a 64Mi PV.
 ** Each {KIE_SERVER} deployment by default requires one 1Gi PV for the database. You can change the database PV size in the template parameters. If you deploy multiple separate {KIE_SERVERS}, each of the servers requires a separate database PV. This requirement does not apply if you use an external database server.
 * Your OpenShift environment supports persistent volumes with `ReadWriteMany` mode. For information about access mode support in OpenShift Online volume plug-ins, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/architecture/#pv-access-modes[Access Modes].
++
+[IMPORTANT]
```
Comment (sterobin) [inline assemblies/assembly_openshift-freeform/main.adoc]: @mramendi I didn't take the time to look at every rendered output, just a couple, but I did notice in just those few that this ReadWriteMany not is aligned with the bullet here but not in the other outputs. I think aligning with the bullet is correct, and I suggest updating all other instances to be the same way.

**Comment by mramendi** [comment]
@errantepiphany but that is what we are doing for OpenShift 3.x right? Operators are supported only on 4.1+?

**Comment by errantepiphany** [comment]
@mramendi You are correct. Sorry; I was out of context here. :)

**Pull request merged**